app/controllers/tweetscontroller.rb

```diff
@@ -0,0 +1,85 @@
+class TweetsController < ApplicationController
+
+    
+
+    # get '/show' do
+    #     @tweets = Tweet.all
+    #     erb :'/show'
+    # end
+
+    get '/tweets/new' do
+        if logged_in?
+             erb :'tweets/new'
+        else # logged out and doesn't let them view the create tweet
+
+        redirect to "/login"
+        end
+    end
+
+     post '/tweets' do
+        if params[:content] == "" #blank tweet
+            redirect to "/tweets/new"
+        else
+             @tweet = current_user.tweets.create(content: params[:content])
+             redirect to "/tweets/#{@tweet.id}"
+        end
+    end
+
+
+    get '/users/:slug' do
+        @tweets = Tweet.all
+        erb :'/tweets'
+    end
+
+    get '/tweets/:id' do
+        if logged_in?
+        @tweet = Tweet.find_by_id(params[:id])
+        erb :"/tweets/show"
+        else
+        redirect to '/login'
+        end
+    end
+
+   
+
+    get '/tweets/:id/edit' do
+        if logged_in?
+        @tweet = Tweet.find_by(params[:id])
```

Comment:
Shouldn't you verify a tweet belongs to the `current_user` (and not just that any user is logged in) before allowing them to edit?